lib/rucio/core/credential.py

```diff
@@ -134,8 +135,8 @@ def get_signed_url(service, operation, url, lifetime=600):
         if colon >= 0:
             host = host[:colon]
 
-        # look up in RSE account configuration by <hostname>_<bucketname>
-        cred_name = host + "_" + bucket
+        # look up in RSE account configuration by RSE ID
+        cred_name = rse
```

Comment:
shouldn't this variable be called `rse_id` ?

Reply:
I have now changed the name to `rse_id` for consistency with the rest of the code.